src/main/kotlin/io/csbroker/apiserver/controller/v1/post/PostControllerV1.kt

```diff
@@ -0,0 +1,72 @@
+package io.csbroker.apiserver.controller.v1.post
+
+import io.csbroker.apiserver.auth.LoginUser
+import io.csbroker.apiserver.common.util.getEmailFromSecurityContextHolder
+import io.csbroker.apiserver.controller.v1.post.request.CommentCreateRequestDto
+import io.csbroker.apiserver.controller.v1.post.request.PostCreateRequestDto
+import io.csbroker.apiserver.controller.v1.post.response.PostResponseDto
+import io.csbroker.apiserver.dto.common.ApiResponse
+import io.csbroker.apiserver.service.post.CommentService
+import io.csbroker.apiserver.service.post.PostService
+import org.springframework.security.core.userdetails.User
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class PostControllerV1(
+    private val postService: PostService,
+    private val commentService: CommentService,
+) {
+    @PostMapping("/api/v1/posts")
+    fun createPost(
+        @LoginUser loginUser: User,
+        @RequestBody postCreateRequestDto: PostCreateRequestDto,
+    ): ApiResponse<Long> {
+        val postId = postService.create(
+            postCreateRequestDto.problemId,
+            postCreateRequestDto.content,
+            loginUser.username,
+        )
+        return ApiResponse.success(postId)
+    }
+
+    @DeleteMapping("/api/v1/posts/{id}")
+    fun deletePost(
+        @LoginUser loginUser: User,
+        @PathVariable("id") id: Long,
+    ): ApiResponse<Unit> {
+        postService.deleteById(id, loginUser.username)
+        return ApiResponse.success()
+    }
+
+    @GetMapping("/api/v1/problem/{problemId}/posts")
+    fun findById(
```

Comment:
```suggestion
    fun findByProblemId(
```
PostController에서 findById는 post의 id를 이용해 검색할 것 으로 유추되는데 여기는 problemId를 이용하는거네요!
조금 더 명확하게 가져가는게 좋아보여요 :)